dll/win32/shell32/shellrecyclebin/recyclebin_v5_enumerator.cpp

```diff
@@ -94,13 +93,29 @@ STDMETHODIMP_(ULONG) RecycleBin5File::Release()
     return refCount;
 }
 
+STDMETHODIMP RecycleBin5File::GetInfo(PDELETED_FILE_INFO pInfo)
+{
+    HRESULT hr = S_OK;
+    ULARGE_INTEGER uli;
+    if (FAILED(GetLastModificationTime(&pInfo->LastModification)))
+        ZeroMemory(&pInfo->LastModification, sizeof(pInfo->LastModification));
+    pInfo->DeletionTime = m_deletedFile.DeletionTime;
+    C_ASSERT(sizeof(pInfo->FileSize) <= sizeof(UINT));
```

Comment:
It's here because this code uses `uli.LowPart`. I don't know if Windows 10/11/12? supports deleted files larger than 4 gib but when that happens, this needs to be updated here.